### PR TITLE
Update applicant program list view

### PR DESCRIPTION
**Pull request by AWBaumann**
### Description
Implemented a basic style/layout based on UX Mocks.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Mocks
image

[Tailwind Play Link](https://play.tailwindcss.com/8yIsUa6rzp?layout=preview)

### Screenshot
image

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java [hunk 1]
```diff
@@ -44,27 +47,160 @@ public Content render(
...
+    body.with(
+        nav()
+            .withClasses(
+                Styles.PT_8,
+                Styles.PB_4,
+                Styles.MB_12,
+                Styles.FLEX,
+                Styles.ALIGN_MIDDLE,
+                Styles.BORDER_B_4,
+                Styles.BORDER_WHITE)
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java:65]: question: will we eventually put these in some kind of named style, like how you did with the admin nav?

**Review by carolinedanzi — APPROVED** [review]
Looks good to me! But I am not a UI expert

File: universal-application-tool-0.0.1/app/views/style/BaseStyles.java [hunk 2]
```diff
@@ -2,6 +2,22 @@
 
 public final class BaseStyles {
 
+  public static final String ADMIN_NAV_BAR =
```
Comment (natsid) [inline universal-application-tool-0.0.1/app/views/style/BaseStyles.java:5]: Should we put this in a file called `AdminStyles.java` similar to the above file? Or if it's used in both views, can rename it `NAV_BAR`.

**Review by natsid — APPROVED** [review]
Looks so good!

File: universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java (hunk 1 above)
Comment (AWBaumann) [reply universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java:65]: Yes, but I tend to wait until things normalize a bit before factoring out the styles. (I just assume that things will change 2-3 times before settling.)

File: universal-application-tool-0.0.1/app/views/style/BaseStyles.java (hunk 2 above)
Comment (AWBaumann) [reply universal-application-tool-0.0.1/app/views/style/BaseStyles.java:5]: Yeah, still kind of waiting for things to be more concrete before trying to really factor out styles.

**Pull request merged**